ci-operator/config/codeready-toolchain/member-operator/codeready-toolchain-member-operator-master.yaml

```diff
@@ -4,12 +4,12 @@ build_root:
 releases:
   initial:
     integration:
-      name: "4.15"
+      name: "4.17"
```

Comment:
did we miss the upgrade to 4.16 here before?

Reply:
Yep, we missed it when were switching to 4.16 last time.